jenkins nodejs12 agent build failing due to incompatible chrome package with centos 7

Build of jenkins nodejs12 agent is failing due to incompatible chrome package with centos 7.

This error:
```
03:45:22      amazon-ebs: Error: Package: google-chrome-stable-95.0.4638.54-1.x86_64 (/google-chrome-stable_current_x86_64)
03:45:22      amazon-ebs:            Requires: libc.so.6(GLIBC_2.18)(64bit)
```
caused the E2E Test "TestVerifyOdsNamespaceImages" to fail:
```
04:33:08      amazon-ebs: === RUN   TestVerifyOdsNamespaceImages
04:33:08      amazon-ebs:     resources.go:31: Image tag 'jenkins-agent-nodejs12:latest' not found.
04:33:08      amazon-ebs: --- FAIL: TestVerifyOdsNamespaceImages (0.02s)```
```

It looks like the problem is that the latest chrome version is not compatible with centos 7, because it doesn't include the GLIBC  2.18.

As fix, I propose to update the jenkins nodejs12 agent to not download the latest version but a specific chrome version.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -30,6 +30,7 @@
 
 ### Fixed
 
+- jenkins nodejs12 agent build failing due to incompatible chrome package with centos 7 ([#656](https://github.com/opendevstack/ods-quickstarters/pull/656))
 - ds-rshiny cleanup cloudera dependency ([#540](https://github.com/opendevstack/ods-quickstarters/pull/540))
 - Removed forcing eslint configuration as it is default ([#573](https://github.com/opendevstack/ods-quickstarters/pull/578))
 - Recover be-python-flask ([#583](https://github.com/opendevstack/ods-quickstarters/issues/583))

--- a/common/jenkins-agents/nodejs12/docker/Dockerfile.centos7
+++ b/common/jenkins-agents/nodejs12/docker/Dockerfile.centos7
@@ -19,19 +19,20 @@ ENV NODEJS_VERSION=12 \
     ENV=/usr/local/bin/scl_enable \
     PROMPT_COMMAND=". /usr/local/bin/scl_enable" \
     LANG=en_US.UTF-8 \
-    LC_ALL=en_US.UTF-8
+    LC_ALL=en_US.UTF-8 \
+    CHROME_VERSION=94.0.4606.81
 
 # install google-chrome (for angular)
 RUN yum-config-manager --enable rhel-7-server-extras-rpms && \
     yum-config-manager --enable rhel-7-server-optional-rpms
 
-ADD https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm /root/google-chrome-stable_current_x86_64.rpm
+ADD https://dl.google.com/linux/chrome/rpm/stable/x86_64/google-chrome-stable-${CHROME_VERSION}-1.x86_64.rpm /root/google-chrome.rpm
 RUN yum -y install redhat-lsb libXScrnSaver
-RUN yum -y install /root/google-chrome-stable_current_x86_64.rpm && \
+RUN yum -y install /root/google-chrome.rpm && \
     ln -s /usr/lib64/libOSMesa.so.8 /opt/google/chrome/libosmesa.so && \
     yum clean all && \
-    dbus-uuidgen > /etc/machine-id
-
+    dbus-uuidgen > /etc/machine-id && \
+    rm /root/google-chrome.rpm
 
 # Install cypress dependencies
 # Please note: xorg-x11-server-Xvfb is not available on RHEL via yum anymore, so "RUN yum install -y xorg-x11-server-Xvfb" won't work.

